ide/celery_app.py

```diff
@@ -5,11 +5,11 @@
 
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'settings')
-app = Celery('ide')
+app = Celery('app', broker='redis://redis:6379/0', backend='redis://redis:6379/0', include=['ide.tasks'])
```

Comment:
It would be a good idea to pick these values from the environment variable and not hard code them directly into the codebase.

Reply:
:+1:  Will do this change once docker works.